### PR TITLE
Add level set plane function

### PR DESCRIPTION
**Pull request by jiangwen84**
ref #15167

### CODE REVIEW

**Comment by jiangwen84** [comment]
@aeslaughter could you review this PR? Thanks.

File: modules/level_set/src/functions/LevelSetOlssonBubble.C [hunk 1]
```diff
@@ -50,7 +50,7 @@ LevelSetOlssonBubble::gradient(Real /*t*/, const Point & p) const
   for (unsigned int i = 0; i < LIBMESH_DIM; ++i)
   {
     g_prime = (p(i) - _center(i)) / (_epsilon * norm);
-    output(i) = (g_prime * std::exp(g)) / ((std::exp(g) + 1) * (std::exp(g) + 1));
+    output(i) = -(g_prime * std::exp(g)) / ((std::exp(g) + 1) * (std::exp(g) + 1));
```
Comment (aeslaughter) [inline modules/level_set/src/functions/LevelSetOlssonBubble.C:53]: Was this just a mistake by me?

File: modules/level_set/test/tests/functions/olsson_plane/tests [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Tests]
+  issues = '#15167'
+  design = 'LevelSetOlssonPlane.md'
+  [./test]
+    type = Exodiff
+    input = olsson_plane.i
+    exodiff = olsson_plane_out.e
+    requirement = "The level set module shall include the plane function defined in [cite!olsson2005conservative]."
+  [../]
```
Comment (aeslaughter) [inline modules/level_set/test/tests/functions/olsson_plane/tests:9]:
```suggestion
  [test]
    type = Exodiff
    input = olsson_plane.i
    exodiff = olsson_plane_out.e
    requirement = "The level set module shall include the plane function defined in [cite!olsson2005conservative]."
  []
```

**Review by aeslaughter — DISMISSED** [review]

File: modules/level_set/src/functions/LevelSetOlssonBubble.C (hunk 1 above)
Comment (jiangwen84) [reply modules/level_set/src/functions/LevelSetOlssonBubble.C:53]: I think so :-)  We probably never use this gradient.

**Review by aeslaughter — APPROVED** [review]

**Pull request merged**